diff -Nurp grub4dos.orig/stage2/asm.S grub4dos/stage2/asm.S
--- grub4dos.orig/stage2/asm.S	2008-05-02 20:00:57.000000000 +0800
+++ grub4dos/stage2/asm.S	2008-05-21 03:07:04.000000000 +0800
@@ -122,7 +122,7 @@ VARIABLE(version_string)
 	.string VERSION
 VARIABLE(config_file)
 #ifndef STAGE1_5
-	.string "/boot/grub/menu.lst"
+	.string "/GEEXBOX/boot/menu.lst"
 #else   /* STAGE1_5 */
 	.long	0xffffffff
 	.string "/boot/grub/stage2"
diff -Nurp grub4dos.orig/stage2/grldrstart.S grub4dos/stage2/grldrstart.S
--- grub4dos.orig/stage2/grldrstart.S	2008-03-14 19:20:12.000000000 +0800
+++ grub4dos/stage2/grldrstart.S	2008-05-21 03:04:05.000000000 +0800
@@ -1294,7 +1294,7 @@
 //	.byte	0		/* mark the end of ascii zero string */
 
 default_config_file:
-	.ascii	"/menu.lst"
+	.ascii	"/GEEXBOX/boot/menu.lst"
 
 	.byte	0		/* mark the end of ascii zero string */
 #endif	/* ! defined(GRLDR_MBR) && (! defined(GRLDR_INSTALL)) */
@@ -1368,7 +1368,7 @@ Entry_32:
 	. = Entry_32 + 0x03
 
 #ifdef	BOOTGRUB
-	.ascii	"GRLDR   "	/* OEM name string (of OS which formatted the disk). */
+	.ascii	"GXLDR   "	/* OEM name string (of OS which formatted the disk). */
 #endif
 
 	. = Entry_32 + 0x0b
@@ -1897,7 +1897,7 @@ msg_BootError_32:
 filename_32:
 
 #ifdef	BOOTGRUB
-	.ascii	"GRLDR      \0"
+	.ascii	"GXLDR      \0"
 #else
 	.ascii	"KERNEL  SYS\0"
 #endif
@@ -2069,7 +2069,7 @@ Entry_12_16:
 	. = Entry_12_16 + 0x03
 
 #ifdef	BOOTGRUB
-	.ascii	"GRLDR   "
+	.ascii	"GXLDR   "
 #endif
 
 	. = Entry_12_16 + 0x0b
@@ -2742,7 +2742,7 @@ msg_BootError_12_16:
 filename_12_16:
 
 #ifdef	BOOTGRUB
-	.ascii	"GRLDR      \0"
+	.ascii	"GXLDR      \0"
 #else
 	.ascii	"KERNEL  SYS\0"
 #endif
@@ -2841,7 +2841,7 @@ Entry_ext2:
 
 #if 0
 
-	.ascii	"ext2 grldr"
+	.ascii	"ext2 gxldr"
 
 #else
 
@@ -3559,7 +3559,7 @@ msg_No_grldr_ext2:
 	.ascii	"No "
 
 filename_ext2:
-	.ascii	"grldr\0"
+	.ascii	"gxldr\0"
 
 	. = Entry_ext2 + 0x1ee
 
@@ -4059,7 +4059,7 @@ msg_NTFS_Not_Found_Error:
 	.ascii "No "
 
 nt_boot_image:
-	.ascii "grldr\0"
+	.ascii "gxldr\0"
 
 // Kernel load address, located at 0x1E8
 	. = Entry_nt + 0x1e8
